src/openforms/js/components/admin/form_design/registrations/zgw/OptionsForm.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Field from 'components/admin/forms/Field';
+
+import ZGWFormFields from './OptionsFormFields';
+
+const ZGWOptionsForm = ({index, name, label, schema, formData, onChange}) => {
+  return (
+    <Field name={name} label={label} errors={[]}>
```

Comment:
I suspect that here we pass an empty array to the errors because the errors are then handled inside the `ZGWFormFields` component explicitly? If yes, maybe worth adding a comment.

I also wonder: is this component needed? Could maybe this component contain directly all the implementation of `ZGWFormFields`?